demos/vc_issuer/tests/issue_credential.rs

```diff
@@ -349,6 +352,7 @@ fn should_return_derivation_origin() {
 }
 
 #[test]
+#[cfg_attr(feature = "exclude_custom_origin", ignore)]
```

Comment:
Integration tests are currently only run on the "normal" version _with_derivation origin support. Do we need the switches here? It would also eliminate the `allow(unused)` above.

Reply:
Good point, we run these test on a wasm with the feature, so no need to ignore them. Removed now the annotations, incl.  the `unused`-one.